collectd-marathon/20-collectd-marathon.conf

```diff
@@ -7,7 +7,9 @@
   Import "marathon"
   LogTraces true
   <Module "marathon">
-    host  ["localhost", "8080", "username", "password"]
+    # The last config is optional. It is "false" by default. Set it to "true" when
+    # operating DC/OS in strict mode
+    host  ["http","localhost", "8080", "username", "password","false"]
```

Comment:
minor nit, but there should be a space between the comma and the quote before localhost and a space between the comma and quote before false